io.github.dxx_rebirth.d2x-rebirth.yml

```diff
@@ -0,0 +1,63 @@
+app-id: io.github.dxx_rebirth.d2x-rebirth
```

Comment:
This ID won't work. There is no repository at https://github.com/dxx-rebirth/d1x-rebirth

Can you change it to something like `io.github.dxx_rebirth.dxx_rebirth.d2x-rebirth` ? The last `d2x-rebirth` portion does not matter, it can be anything.

See the control over domain portion here https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain